src/test/java/cms/gov/madie/measure/utils/QiCoreJsonUtilTest.java

```diff
@@ -488,8 +499,107 @@ public void testGetTestCaseGroupPopulationsFromMeasureReportNoCount()
l\":\"Encounter/encounter-inpatient-1c2a\",\"resource\":{\"id\":\"encounter-inpatient-1c2a\",\"resourceType\":\"Encounter\",\"meta\":{\"profile\":[\"http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-encounter\"]},\"status\":\"finished\",\"class\":{\"system\":\"http://terminology.hl7.org/CodeSystem/v3-ActCode\",\"code\":\"IMP\",\"display\":\"inpatient\"},\"type\":[{\"coding\":[{\"system\":\"http://snomed.info/sct\",\"version\":\"2022-03\",\"code\":\"183452005\",\"display\":\"Emergency hospital admission (procedure)\",\"userSelected\":true}]}],\"period\":{\"start\":\"2024-01-01T00:01:00.000+00:00\",\"end\":\"2024-01-02T08:30:00.000+00:00\"},\"hospitalization\":{\"dischargeDisposition\":{\"coding\":[{\"system\":\"http://snomed.info/sct\",\"code\":\"183919006\",\"display\":\"Urgent admission to hospice (procedure)\",\"userSelected\":true}]}},\"subject\":{\"reference\":\"Patient/0e3be52f-723e-4df4-a584-337daa19e259\"}}},{\"fullUrl\":\"MedicationRequest1/schedule-ii-iii-opioid-medications-1c2b\",\"resource\":{\"id\":\"schedule-ii-iii-opioid-medications-1c2b\",\"resourceType\":\"MedicationRequest\",\"meta\":{\"profile\":[\"http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-medicationrequest\"]},\"status\":\"active\",\"intent\":\"order\",\"doNotPerform\":false,\"category\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/medicationrequest-category\",\"code\":\"discharge\",\"display\":\"Discharge\",\"userSelected\":true}]}],\"medicationCodeableConcept\":{\"coding\":[{\"system\":\"http://www.nlm.nih.gov/research/umls/rxnorm\",\"code\":\"1014599\",\"display\":\"acetaminophen 300 MG / oxycodone hydrochloride 10 MG Oral Tablet\",\"userSelected\":true}]},\"authoredOn\":\"2024-01-02T08:30:00.000+00:00\",\"requester\":{\"reference\":\"Practitioner/f007\",\"display\":\"Patrick Pump\"},\"subject\":{\"reference\":\"Patient/0e3be52f-723e-4df4-a584-337daa19e259\"}}},{\"fullUrl\":\"Coverage/1\",\"resource\":{\"resourceType\":\"Coverage\",\"beneficiary\":{\"reference\":\"Patient/0e3be52f-723e-4df4-a584-337daa19e259\"},\"id\":\"1\",\"meta\":{\"profile\":[\"http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-coverage\"]},\"payor\":[{\"reference\":\"Organization/123456\"}],\"status\":\"active\"}},{\"fullUrl\":\"Organization/123456\",\"resource\":{\"resourceType\":\"Organization\",\"active\":true,\"address\":[{\"use\":\"billing\",\"type\":\"postal\",\"line\":[\"P.O. Box 660044\"],\"city\":\"Dallas\",\"state\":\"TX\",\"postalCode\":\"75266-0044\",\"country\":\"USA\"}],\"id\":\"123456\",\"identifier\":[{\"use\":\"temp\",\"system\":\"urn:oid:2.16.840.1.113883.4.4\",\"value\":\"21-3259825\"}],\"meta\":{\"profile\":[\"http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-organization\"]},\"name\":\"Blue Cross Blue Shield of Texas\",\"telecom\":[{\"system\":\"phone\",\"value\":\"(+1) 972-766-6900\"}],\"type\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/organization-type\",\"code\":\"pay\",\"display\":\"Payer\"}]}]}}]}";
```

Comment:
Do you think we can add these multiple test JSON strings as a resource and then load them ?

Reply:
@RohitKandimalla Done as well, added the resource test json files.

Reply:
I could still see this

Reply:
@RohitKandimalla Added another resource file for this. Thanks Rohit!